bin/tools/Templates/FrontendUnitTest.js

```diff
@@ -665,6 +903,123 @@ export class FrontendUnitTest extends AbstractTemplate {
    * @returns {string}
    * @constructor
    */
+  static get MODELS_TEST_TPL() {
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for such strings, instead of array.join(os.EOL)